weather/weather_forecast.py

```diff
@@ -19,68 +18,21 @@ def retrieve_data_about_the_weather_forecast():
     """
     weather_forecast_text = ''
 
-    BASE_URL = "http://api.openweathermap.org/data/2.5/weather?"
-    CITY = "Sofia"
+    base_url = "http://api.openweathermap.org/data/2.5/weather?"
+    city = "Sofia"
```

Comment:
We should make this variable to be dynamic and Dancho to read the weather based on the user input.

Reply:
Fixed with https://github.com/Stake-And-Rope/dancho-voice-assistant/commit/44c51aee3a67b83a0086dce4c0cf4081f91466d4